edxval/api.py

```diff
@@ -355,6 +356,7 @@ def get_videos_for_course(
         {"courses__course_id": unicode(course_id), "courses__is_hidden": False},
         sort_field,
         sort_dir,
+        context={'course_id': course_id}
```

Comment:
using single quotes is better, we should always use them whenever we can.  May be update above quotes.